src/compiler/checker.ts

```diff
@@ -35661,6 +35665,8 @@ namespace ts {
                 case ExternalEmitHelpers.MakeTemplateObject: return "__makeTemplateObject";
                 case ExternalEmitHelpers.ClassPrivateFieldGet: return "__classPrivateFieldGet";
                 case ExternalEmitHelpers.ClassPrivateFieldSet: return "__classPrivateFieldSet";
+                case ExternalEmitHelpers.CreateBinding: return "__createBinding";
+                case ExternalEmitHelpers.SetModuleDefault: return "__setModuleDefault";

```

Comment:
Just noticed this: This doesn't need to be checked since its an internal implementation detail of `__importStar`.

Reply:
https://github.com/microsoft/TypeScript/pull/36998